auction-app-frontend/src/services/productService.js

```diff
@@ -15,15 +15,15 @@ const getProductRandom = () => {
   return getRequest("/products/random");
 };
 
-const getProducts = (page, size, categoryId, searchQuery) => {
+const getProducts = (page, size, categoryId, searchQuery, sortCriteria, sortDirection) => {
   // construct query string based on the presence of categoryId or search query
-  const categoryParam = categoryId ? `&category_id=${ categoryId }` : "";
+  const categoryParam = categoryId ? `&categoryId=${ categoryId }` : "";
```

Comment:
Haven't we agreed that url shouldn't be in the camel case
We should follow this format `url-second-part`